avatar/avatar.css

```diff
@@ -0,0 +1,63 @@
+/* avatar CSS Start Here */
+
+.avatar {
+  color: white;
+  font-size: 2.5rem;
+  font-weight: 600;
+  cursor: pointer;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  height: 4rem;
+  width: 4rem;
+  border-radius: 50%;
+  background-color: var(--grey-color);
+  text-transform: uppercase;
+  white-space: nowrap;
+}
+
+.avatar-box {
+  display: inline-block;
+  width: 4rem;
+  height: 4rem;
+  margin: 0;
+}
+
+.avatar-img {
+  height: 100%;
+  border-radius: 100%;
+}
+
+.avatar-img {
+  transition: transform 0.2s;
+}
+
+.avatar-img:hover {
+  transform: scale(1.5);
+}
+
+.sm {
+  font-size: 1.5rem;
+  width: 2.5rem;
+  height: 2.5rem;
+}
+
+.md {
+  font-size: 3rem;
+  width: 4rem;
+  height: 4rem;
+}
+
+.lg {
+  font-size: 4rem;
+  width: 6rem;
+  height: 6rem;
+}
```

Comment:
Can you please explain what these classes mean? Is it related to text font?

Reply:
it is related to size of the box container avatar and font size related to the respective box size.

Reply:
So it would be better to give .avatar-sm , .avatar-md , .avatar-lg that would sound more meaningful

Reply:
i'm trying to use short the name but yes .avatar-sm is more meaningful than simple sm.

Reply:
Do the changes as suggested, otherwise looks good 😊

Reply:
thank you 😊